src/views/Rules/ListRuleRevisions/index.jsx

```diff
@@ -48,9 +48,7 @@ function ListRuleRevisions(props) {
   const [fetchedRevisions, fetchRevisions] = useAction(getRevisions);
   const addSC = useAction(addScheduledChange)[1];
   const { ruleId } = props.match.params;
-  // eslint-disable-next-line prefer-destructuring
-  const error = fetchedRevisions.error;
-  const isLoading = fetchedRevisions.loading;
+  const { error, loading: isLoading } = fetchedRevisions;
```

Comment:
This was actually intended so that if we ever add new requests to this view, we could just do

```jsx
const error = fetchedRevisions.error || ...
const isLoading = fetchedRevisions.loading || ...;
```

Reply:
Ah, let's maintain this, in that case.